test/integration/scheduler/scheduler_test.go

```diff
@@ -103,6 +103,7 @@ var _ = ginkgo.Describe("Scheduler", func() {
 		prodBEClusterQ = testing.MakeClusterQueue("prod-be-cq").
```

Comment:
MakeClusterQueue uses StrictFIFO by default. We should make it match the API default.

Reply:
+1, and we should have a test for StrictFIFO testing that a workload continues to be blocked if another fronting it is inadmissible.

Reply:
> MakeClusterQueue uses StrictFIFO by default. We should make it match the API default.

Added.